pkg/scheduler/metrics/metrics.go

```diff
@@ -309,9 +312,18 @@ func InitMetrics() {
 		[]string{"result"})
 
 	CacheSize = metrics.NewGaugeVec(
+		&metrics.GaugeOpts{
+			Subsystem:         SchedulerSubsystem,
+			Name:              "scheduler_cache_size",
+			Help:              "Number of nodes, pods, and assumed (bound) pods in the scheduler cache.",
+			StabilityLevel:    metrics.ALPHA,
+			DeprecatedVersion: "1.33.0",
+		}, []string{"type"})
+
+	SchedulerCacheSize = metrics.NewGaugeVec(
```

Comment:
I meant, can we swap the variable names?
`CacheSize` for "cache_size" (new one) and `SchedulerCacheSize` for "scheduler_cache_side" (old one).

Reply:
got this.

Reply:
If we modify the variable names, there seems to be a hint problem 🤔 

```bash
ERROR: pkg/scheduler/backend/cache/cache.go:761:2: SA1019: metrics.SchedulerCacheSize is deprecated: SchedulerCacheSize is deprecated, and will be removed at v1.33. Please use CacheSize instead. (staticcheck)
ERROR: 	metrics.SchedulerCacheSize.WithLabelValues("assumed_pods").Set(float64(len(cache.assumedPods)))
ERROR: 	^
ERROR: pkg/scheduler/backend/cache/cache.go:762:2: SA1019: metrics.SchedulerCacheSize is deprecated: SchedulerCacheSize is deprecated, and will be removed at v1.33. Please use CacheSize instead. (staticcheck)
ERROR: 	metrics.SchedulerCacheSize.WithLabelValues("pods").Set(float64(len(cache.podStates)))
ERROR: 	^
ERROR: pkg/scheduler/backend/cache/cache.go:763:2: SA1019: metrics.SchedulerCacheSize is deprecated: SchedulerCacheSize is deprecated, and will be removed at v1.33. Please use CacheSize instead. (staticcheck)
ERROR: 	metrics.SchedulerCacheSize.WithLabelValues("nodes").Set(float64(len(cache.nodes)))
ERROR: 	
```

Reply:
Try putting this:

```go
//nolint: staticcheck // we intentionally keep them with the deprecation and will remove at v1.33.
```